Src/GoogleApis.Auth/OAuth2/AuthorizationCodeInstalledApp.cs

```diff
@@ -88,7 +87,18 @@ public ICodeReceiver CodeReceiver
                     taskCancellationToken).ConfigureAwait(false);
             }
 
-            return new UserCredential(flow, userId, token);
+            return new UserCredential(Flow, userId, token);

```

Comment:
Fixed. I have no recollection of making this particular change, and if it weren't for the evidence in front of my face I'd swear you were trying to pull one over on me...